packages/cli/src/commands/build.js

```diff
@@ -39,7 +40,7 @@ export const handler = async ({
 }) => {
   const { base: BASE_DIR } = getPaths()
 
-  if (app.includes('api')) {
+  if (app.includes('api') && prismaExists) {
```

Comment:
Maybe we should add this check to `generatePrismaClient`

Reply:
see latest